numba/core/overload_glue.py

```diff
@@ -27,28 +27,28 @@ def __init__(self, function, typing_key=None):
         # is all lazy.
         self._build()
 
-    def _stub_generator(self, nargs, body_func, kwargs=None):
-        """This generates a function that takes "nargs" count of arguments
-        and the presented kwargs, the "body_func" is the function that'll
+    def _stub_generator(self, body_func, varnames_gen):
+        """This generates a function based on the argnames generated by the
+        varnames_gen generator, the "body_func" is the function that'll
```

Comment:
```suggestion
        """This generates a function based on the argnames provided by
        varnames, the "body_func" is the function that'll
```

Reply:
Thanks, fixed these and the side-effects of fixing these in 2e833f7